app/charts/shared/axis-height-linear.tsx

```diff
@@ -55,9 +58,23 @@ export const AxisHeightLinear = () => {
   return (
     <>
       <g>
-        <text x={0} y={0} dy={labelFontSize} fontSize={labelFontSize}>
-          {yAxisLabel}
-        </text>
+        {yAxisDescription ? (
+          <Tooltip arrow title={yAxisDescription}>
+            <text
+              x={0}
+              y={0}
+              dy={labelFontSize}
+              fontSize={labelFontSize}
+              textDecoration="underline"
+            >
+              {yAxisLabel}
+            </text>
+          </Tooltip>
+        ) : (
+          <text x={0} y={0} dy={labelFontSize} fontSize={labelFontSize}>
+            {yAxisLabel}
+          </text>
+        )}
```

Comment:
Nit: A MaybeTooltip component that would do the conditional wrapping based on whether `title` exists could be cleaner since we would be sure that text elements are changed at the same time.

Reply:
I have added it, thanks for suggestion 🙇 Is it fine like this? I also thought about having a prop like `wrapper=((children) => ...)` on the MaybeTooltip, but then I think we would need a useCallback for each case like this to prevent re-rendering, which would also not be great I think 😅

Reply:
Yes I think this is fine ! Thanks !